source/aggregation/agg-exp-ops.txt

```diff
@@ -0,0 +1,1183 @@
+   aggregation, you must define the entire expression using the BSON ``Document``
+   type.
+   
+.. TODO add to note To learn more about the ``Document`` type, see :ref:`<kotlin-sync-document-format>`.
+
+Arithmetic Operations
+~~~~~~~~~~~~~~~~~~~~~
+
+You can perform an arithmetic operation on a value of type ``MqlInteger`` or
+``MqlNumber`` using the methods described in this section.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Method
+     - Aggregation Pipeline Operator
+
+   * - | `abs() for MqlInteger <{+core-api+}/com/mongodb/client/model/mql/MqlInteger.html#abs()>`__
+       | `abs() for MqlNumber <{+core-api+}/com/mongodb/client/model/mql/MqlNumber.html#abs()>`__
+
+     - :manual:`$abs </reference/operator/aggregation/abs/>`
+
+   * - | `add() for MqlInteger <{+core-api+}/com/mongodb/client/model/mql/MqlInteger.html#add(int)>`__
+       | `add() for MqlNumber <{+core-api+}/com/mongodb/client/model/mql/MqlNumber.html#add(com.mongodb.client.model.mql.MqlNumber)>`__
+
+     - :manual:`$add </reference/operator/aggregation/add/>`
+
+   * - `divide() <{+core-api+}/com/mongodb/client/model/mql/MqlNumber.html#divide(com.mongodb.client.model.mql.MqlNumber)>`__
+     - :manual:`$divide </reference/operator/aggregation/divide/>`
+
+   * - | `multiply() for MqlInteger <{+core-api+}/com/mongodb/client/model/mql/MqlInteger.html#multiply(int)>`__
+       | `multiply() for MqlNumber <{+core-api+}/com/mongodb/client/model/mql/MqlNumber.html#multiply(com.mongodb.client.model.mql.MqlNumber)>`__
+
+     - :manual:`$multiply </reference/operator/aggregation/multiply/>`
+
+   * - `round() <{+core-api+}/com/mongodb/client/model/mql/MqlNumber.html#round()>`__
+     - :manual:`$round </reference/operator/aggregation/round/>`
+
+   * - | `subtract() for MqlInteger <{+core-api+}/com/mongodb/client/model/mql/MqlInteger.html#subtract(int)>`__
+       | `subtract() for MqlNumber <{+core-api+}/com/mongodb/client/model/mql/MqlNumber.html#subtract(com.mongodb.client.model.mql.MqlNumber)>`__
+
+     - :manual:`$subtract </reference/operator/aggregation/subtract/>`
+
+Suppose you have weather data for a specific year that includes the
+precipitation measurement (in inches) for each day. You want find the average
+precipitation, in millimeters, for each month.
+
+The ``multiply()`` operator multiplies the ``precipitation`` field by
+``25.4`` to convert the value to millimeters. The ``avg()`` accumulator method
+returns the average as the ``avgPrecipMM`` field. The ``group()`` method
+groups the values by month given in each document's ``date`` field.
+
+The following code shows the pipeline for this aggregation:
+
+.. literalinclude:: /includes/aggregation/aggExpressions.kt
+   :start-after: start-arithmetic-aggregation
+   :end-before: end-arithmetic-aggregation
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+The following code provides an equivalent aggregation pipeline in the
+Query API:
```

Comment:
Would this be more proper to say? Applies to all instances of this line.

```suggestion
The following code provides an equivalent aggregation pipeline using the
Query API:
```

Reply:
Originally this was meant to say "in MQL" before the terminology changed. I think "in" is as proper as "that uses" in this context